project/AppDependencies.scala

```diff
@@ -24,11 +23,9 @@ private object AppDependencies {
     "uk.gov.hmrc" %% "play-whitelist-filter" % playAllowListFilterVersion,
     "uk.gov.hmrc" %% "json-encryption" % jsonEncryptionVersion,
     "uk.gov.hmrc" %% "simple-reactivemongo" % playReactivemongoVersion,
-    "uk.gov.hmrc" %% "auth-client" % authVersion,
-    "uk.gov.hmrc" %% "play-ui" % "8.15.0-play-26",
-    "uk.gov.hmrc" %% "bootstrap-frontend-play-26" % "2.25.0",
-    "uk.gov.hmrc" %% "govuk-template" % "5.58.0-play-26",
-    "uk.gov.hmrc" %% "http-verbs" % "11.7.0",
+    "uk.gov.hmrc" %% "play-ui" % "9.4.0-play-26",
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-26" % "5.3.0",
+    "uk.gov.hmrc" %% "govuk-template" % "5.66.0-play-26",
```

Comment:
I think we can use this version if we don't see any compilation issues
`"silencer-plugin" % "1.7.5" `

Reply:
Done